pages/common/tlmgr-search.md

```diff
@@ -0,0 +1,24 @@
+# tlmgr search
+
+> Search for TeX Live packages using (Perl) regular expressions.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- Search the package name and descriptions of all locally installed packages:
```

Comment:
```suggestion
- Search for the package name and descriptions of all locally installed packages from a regular expression:
```

Reply:
I left out the "regular expression" part, because I mentioned in the command description that the command uses regex. Also, it makes the examples a little longer a lot more repetitive. @sbrl any idea how we could write this without it sounding weird?

Reply:
@CleanMachine1 could you please check this?